react/features/toolbox/defaultToolbarButtons.js

```diff
@@ -153,12 +154,32 @@ export default {
         id: 'toolbar_button_dial_out',
         onClick() {
             JitsiMeetJS.analytics.sendEvent('toolbar.sip.clicked');
-            APP.store.dispatch(openDialOutDialog());
+
+            return openDialOutDialog();
         },
         tooltipKey: 'dialOut.dialOut'
     },
 
     /**
+     * The descriptor of the device selection toolbar button.
+     */
+    deviceselection: {
```

Comment:
Hm. I think that we need to use the current Settings button here. It already has some logic that make it open the device selection dialog if (interfaceConfig.SETTINGS_SECTIONS.length === 1 && UIUtil.isSettingEnabled('devices'))